sysutils/hostage/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+
+github.setup        emcrisostomo hostage 1.0.0
+github.tarball_from tarball
+revision            0
```

Comment:
Please swap these two lines so that the `revision` is immediately below the `version` in the `github.setup` line.